packages/admin-ui/src/components/02_atoms/LoadComponent/LoadComponent.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as MaterialUI from '@material-ui/core';
+
+class LoadComponent extends React.Component {
```

Comment:
Would something like `RemoteComponent` or `ExternalComponent` better describe for someone new the purpose of this component?

Reply:
Maybe we could use ```EnsureComponent``` to describe the optionality?

Reply:
I like the proposed name from @dawehner! Do you have any thoughts @mattgrill?

Reply:
I like it. Lets do this.

Reply:
Done